apps/nextra/pages/en/build/guides/exchanges.mdx

```diff
@@ -111,7 +116,8 @@ returns a balance of `100000000`, this is `1 APT`.  If it returns `100` this is
 
 <Callout type="info">
   Note: This includes APT and any other coin that was migrated to a fungible
-  asset.
+  asset.  If the asset is a migrated coin, use this over fungible asset balance.  The
```

Comment:
Might be good to clarify what "this" means

Reply:
Will followup